src/curriculum/dataflow/dataflow.json

```diff
@@ -16,6 +16,12 @@
       "iconId": "icon-text-tool",
       "isTileTool": true
     },
+    {
+      "name": "dataflow",
+      "title": "Dataflow",
+      "iconId": "icon-drawing-tool",
```

Comment:
If we were feeling ambitious we could convert something from DF2 to an SVG icon to be used here.  This would work:
https://github.com/concord-consortium/flow-server/blob/master/static/flow/images/icon-program.png

Not very urgent though and probably better to do in a style pass.